test/e2e/kustomize_test.go

```diff
@@ -141,6 +144,39 @@ func TestKustomizeDeclarativeInvalidApp(t *testing.T) {
 		Expect(Condition(ApplicationConditionComparisonError, "invalid-kustomize/does-not-exist.yaml: no such file or directory"))
 }
 
+<<<<<<< HEAD
+func TestKustomizeBuildOptionsLoadRestrictor(t *testing.T) {
+	out, err := fixture.Run("", "kustomize", "version")
+	if err != nil {
+		assert.Fail(t, "was not able to run `kustomize version`")
+	}
+	// This test is only relevant to Kustomize version 2.1.0+
+	if strings.Contains(out, "KustomizeVersion:2.0.") || strings.Contains(out, "KustomizeVersion:1.0.") {
```

Comment:
hmmm... we don't support k1 or k2 anymore?

Reply:
Yup -- will remove